tasks/jest/setup.js

```diff
@@ -9,6 +9,8 @@
 
 global.__DEV__ = true;
 
+jest.setTimeout(20000);
```

Comment:
Does this work if we put it in the `toHaveNoDAPViolations`matcher instead of our setup file?

Reply:
I would have thought this was part of Jest's setup -- at least that's what the Jest docs implied. That's where I'd look for it at least 🦐